testing/btest/Baseline/coverage.bare-load-baseline/canonified_loaded_scripts.log

```diff
@@ -116,6 +116,17 @@ scripts/base/init-frameworks-and-bifs.zeek
     scripts/base/frameworks/input/readers/sqlite.zeek
   scripts/base/frameworks/analyzer/__load__.zeek
     scripts/base/frameworks/analyzer/dpd.zeek
+    scripts/base/frameworks/analyzer/logging.zeek
+      scripts/base/frameworks/config/__load__.zeek
```

Comment:
If I read this diff correctly, we'll now be loading `frameworks/config` and `frameworks/cluster` in bare mode. This might be okay, but wanted to point it out explicitly.

Reply:
I think that's fine, but maybe should add them them to `base/init-frameworks-and-bifs.zeek` then to make that explicit.